web_src/css/dashboard.css

```diff
@@ -160,3 +160,12 @@
 .feeds .news:last-of-type .divider {
   display: none !important;
 }
+
+.feeds .list ul li.active {
+  background: var(--color-primary);
+  color: var(--color-primary-contrast);
+}
+
+.feeds .list ul li.active svg {
+  color: inherit;
+}
```

Comment:
I think these styles could be moved to `DashboardRepoList.vue`, the scoped style block.

(Other styles have been moved by other PRs)

Reply:
Moved